server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -49,7 +50,7 @@
 
     private static final int SERVER_DEFAULT_PORT = 8091;
     private static final String SERVER_DEFAULT_STORE_MODE = "file";
-    private static final Long SERVER_DEFAULT_NODE = ThreadLocalRandom.current().nextLong(1024);
+    private static final Long SERVER_DEFAULT_NODE = IdWorker.initWorkerId();
```

Comment:
whether the serverNode is null or out of range,do this again

Reply:
Already processed